verification/verify_detectors.py

```
@@ -0,0 +1,95 @@
+        for det in detectors:
+            found = False
+            try:
+                # Use a flexible locator
+                locator = page.get_by_text(det, exact=False).first
+                if locator.count() > 0:
+                    locator.scroll_into_view_if_needed()
+                    if locator.is_visible():
+                        print(f"Found detector: {det}")
+                        found = True
+            except Exception:
+                pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)